scripts/inputs/build_inputs.py

```diff
@@ -50,6 +50,10 @@
 # the run based on test_batch_small does not currently produce results for module03 and beyond because those files depend
 # resources not defined for the small batch (but defined for the large batch).
 #
+# The 'transpose_tsv' function simplifies the editing, viewing of diffs, and tracking of changes in Git for
```

Comment:
I would just add an explicit sentence at the start of this paragraph describing the new behavior, something like "Any template files with a filename that begins with the string 'workspace' will be transposed before the final inputs are written. This functionality allows us to simplify the editing, viewing of diffs, and tracking of changes in Git of workspace TSV files that Terra expects in a two-row format."

Reply:
@cwhelan Thank you for your comment! Following your suggestion, I updated the description in [this commit](https://github.com/broadinstitute/gatk-sv/pull/523/commits/40565a248f0df80ae2968945be8b0fe31149c817).